### ISSUE TITLE
2021-04 Question: Radial Cases

### ISSUE THREAD

**Issue by OPMUSER** [OPM/opm-simulators#3246]
Do we need the COORDSYS keyword item three to be set to COMP to complete the circle?

Option is required by the commercial simulator.

**Comment by goncalvesmachadoc** [comment]
Yes, the keyword COMP is needed. I just checked with Pieter and the complete the circle method is currently not being used. We will fix that.
